src/coreclr/minipal/minipal.h

```diff
@@ -3,6 +3,12 @@
 //
 #include <stddef.h>
 
+#ifdef _MSC_VER
+#define DEBUG_BREAK __debugbreak()
+#else
+#define DEBUG_BREAK DebugBreak()
```

Comment:
`DebugBreak()` is defined in CoreCLR Win32 PAL. minipal should not have a dependency on CoreCLR Win32 PAL. This macro should be in CoreCLR Win32 PAL. Or even - since this seems to be only used in superpmi test tool, define it for superpmi only.

This macro is not very useful. Release build should not contain debug breaks. And asserts provide equivalent for debug builds.

Reply:
Ah, I actually have it defined in `src/coreclr/tools/superpmi/superpmi-shared/standardpch.h`. First put it there to see if it compiles then forgot to revert this file.